documentation/src/main/asciidoc/topics/proc_adding_protostream_annotations.adoc

```diff
@@ -7,6 +7,11 @@ Declare ProtoStream metadata by adding annotations to a Java class and its membe
 .Procedure
 
 . Annotate the Java fields that you want to marshall with `@ProtoField`, either directly on the field or on the getter or setter method.
++
+Any non-annotated fields in your Java class are transient.
+For example, you have a Java class with 15 fields and annotate five of them.
+The resulting schema contains only those five fields and only those five fields are marshalled to {brandname}.
++
```

Comment:
@danberindei @anistor @gustavonalle How about this clarification for transient fields in marshalled classes?

Reply:
@oraNode I don't think it's correct to say "marshalled to {brandname}"

A correct, but much more verbose, description would be "... and only those five fields are marshalled when storing a class instance in {brandname} (if the encoding if anything other than `application/x-java-object`) or when copying the instance to another {brandname} node (if the encoding is `application/x-java-object`)". Maybe "... and only those five fields are marshalled when storing a class instance in {brandname}" is good (and short) enough?

Reply:
thanks @danberindei I think the good enough and short enough description is fine. I'll update with that. that correct/verbose description is a lot to parse for something that is kind of a side note.